### PR TITLE
Complete the intro section of the testing walkthrough. 

### PR DESCRIPTION
**Pull request by AnudeepYakkala**
Modify greet() function to trim white spaces from the name and add an additional test to check if this functionality works as intended.

### CODE REVIEW

File: walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java [hunk 1]
```diff
@@ -30,4 +30,14 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
```
Comment (chen-dawn) [inline walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java:35]: Add tests for other cases: e.g. `\n`, `\t`, doesn't trim whitespace in between words

Comment (AnudeepYakkala) [reply walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java:35]: Okay

File: walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java [hunk 2]
```diff
@@ -22,6 +22,6 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    return "Hello " + name.trim().replaceAll("[\\n\\t]", "");
```
Comment (chen-dawn) [inline walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java:25]: I think you don't need the `replaceAll`, `trim` already takes care of tabs and newlines.

Comment (AnudeepYakkala) [reply walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java:25]: Oh, I thought trim was only spaces. I will remove the replaceAll().

**Pull request merged**